presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -663,29 +661,26 @@ void PeriodicTaskManager::addHttpEndpointLatencyStatsTask() {
 
 void PeriodicTaskManager::addWatchdogTask() {
   addTask(
-      [this,
-       deadlockedTasks = std::vector<std::string>(),
-       opCalls = std::vector<velox::exec::Task::OpCallInfo>()]() mutable {
-        deadlockedTasks.clear();
-        opCalls.clear();
-        if (!taskManager_->getLongRunningOpCalls(
-                stuckDriverThresholdMs_, deadlockedTasks, opCalls)) {
+      [this] {
+        deadlockedTasks_.clear();
```

Comment:
The idea was that we want the memory to be preallocated to reduce the lock duration on the task mutex.  But looking at the code carefully, as soon as they becomes non-empty we detach the worker, so we can keep them as local, since they are always empty in normal case.